packages/packages.csv

```diff
@@ -211,6 +211,7 @@ Microsoft.Extensions.StackTrace.Sources,noship,exclude
 Microsoft.Extensions.TypeNameHelper.Sources,noship,exclude
 Microsoft.Extensions.WebEncoders,ship,include
 Microsoft.Extensions.WebEncoders.Sources,noship,exclude
+Microsoft.MonoDevelop.LanguageServices.Razor,shipoob,exclude
```

Comment:
Is this really the right branding? MonoDevelop?

Reply:
Could be. This assembly is going to have MonoDevelop specific code in it + dependencies that are monodevelop specific; on the other hand it will also have VS for mac specific pieces in it (`Microsoft.VisualStudio.Text`). Up to other suggestions 😄

Reply:
Starting a thread about this

Reply:
Decision was to do `Microsoft.VisualStudio.Mac.LanguageServices.Razor`. PR Updated.